outport/host/driver.go

```diff
@@ -102,6 +114,21 @@ func (o *hostDriver) handleAction(args interface{}, topic string) error {
 	return nil
 }
 
+// RegisterHandlerForSettingsRequest will register the handler function for the settings request
+func (o *hostDriver) RegisterHandlerForSettingsRequest(handlerFunction func()) error {
+	return o.payloadProc.SetHandlerFunc(handlerFunction)
+}
+
+// SetCurrentSettings will send the current settings
+func (o *hostDriver) SetCurrentSettings(config outport.OutportConfig) error {
```

Comment:
I don't quite like this approach. It is too "focused" on a single handle/topic, e.g.: in this case for settings

We could've made this way more general. If we need to send another information we need to always add another 2 funcs per info.

Also, `SetCurrentSettings` **sends** data, not **sets** it

Reply:
Still not solved